test/inspector/inspector-helper.js

```diff
@@ -497,11 +496,12 @@ Harness.prototype.kill = function() {
 
 exports.startNodeForInspectorTest = function(callback,
                                              inspectorFlags = ['--inspect-brk'],
-                                             opt_script_contents) {
+                                             mainScript = 'loop.js') {
   const args = [].concat(inspectorFlags);
-  if (opt_script_contents) {
+  if (!mainScript.endsWith('.js')) {
```

Comment:
This is an undocumented internal function. I think it's safe to assume that whoever wants to add a new test will look at the code to figure out how it works (just like I did).